packages/office-ui-fabric-react/src/components/Icon/FontIcon.tsx

```diff
@@ -11,12 +11,18 @@ export const getIconContent = memoizeFunction((iconName?: string) => {
     code: undefined
   };
 
+  if (!code) {
```

Comment:
Nitpicky thing, but the formatting on this file is off now (indentation of this function's parameters is non-standard, and there's an extra semicolon later), which is odd because Prettier should be running automatically on commit to standardize the formatting. This makes me suspect that your local environment isn't set up properly and/or you're skipping/disabling pre-commit hooks.

Reply:
There was some issue with the setup. Now Prettier is taking care of formatting.